storage/gcp/gcp.go

```diff
@@ -538,42 +643,20 @@ func newSpannerSequencer(ctx context.Context, spannerDB string, maxOutstanding u
 //     This table coordinates integration of the batches of entries stored in
 //     Seq into the committed tree state.
 func (s *spannerSequencer) initDB(ctx context.Context, spannerDB string) error {
-	adminClient, err := database.NewDatabaseAdminClient(ctx)
-	if err != nil {
-		return err
-	}
-	defer adminClient.Close()
-
-	op, err := adminClient.UpdateDatabaseDdl(ctx, &adminpb.UpdateDatabaseDdlRequest{
-		Database: spannerDB,
-		Statements: []string{
+	return createAndPrepareTables(
```

Comment:
Can these refactoring changes to DB init be factored into another PR? They look good enough standalone, and there's already a lot going on here.

Reply:
Good call, done: #477